packages/cli/create-medusa-app/src/commands/create.ts

```diff
@@ -53,6 +53,13 @@ export default async ({
   withNextjsStarter = false,
   verbose = false,
 }: CreateOptions) => {
+  const nodeVersion = getNodeVersion()
+  if (nodeVersion < 20) {
+    logMessage({
+      message: `Medusa requires at least v20 of Node.js. You're using v${nodeVersion}. Please install at least v20 and try again: https://nodejs.org/en/download`,
+      type: "error"
+    })
```

Comment:
q: should we exit here as well?

Reply:
This one exits automatically when type is `error`.